service/controller/v5/resource/blobobject/update.go

```diff
@@ -0,0 +1,124 @@
+package blobobject
+
+import (
+	"context"
+	"fmt"
+	"github.com/giantswarm/azure-operator/service/controller/v5/key"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller"
+)
+
+func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	containerObjectToUpdate, err := toContainerObjectState(updateChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	storageAccountsClient, err := r.getAccountsClient()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	sc := &BlobClient{
+		storageAccountsClient: storageAccountsClient,
+	}
+
+	groupName := key.ClusterID(customObject)
+	storageAccountName := key.StorageAccountName(customObject)
+	containerName := key.BlobContainerName()
+
+	// if there is no storage account - return and wait for deployment to finish storage account operation.
+	_, err = sc.storageAccountsClient.GetProperties(ctx, groupName, storageAccountName)
```

Comment:
That is already done in `GetCurrentState`. What we should do there is canceling the reconciliation for the resource so the rest of the resource is not being executed on such a condition.

Reply:
How can I cancel reconciliation in the `GetCurrentState`?